alacritty_terminal/src/term/mod.rs

```diff
@@ -58,6 +58,8 @@ pub trait Search {
     fn semantic_search_left(&self, _: Point<usize>) -> Point<usize>;
     /// Find the nearest semantic boundary _to the point_ of provided point.
     fn semantic_search_right(&self, _: Point<usize>) -> Point<usize>;
+    /// Find the extent of wrap lines to either side of provided point.
+    fn search_wrapline(&self, _: Point<usize>) -> Span;
```

Comment:
To match the semantic search, I think it's better to use `line_search_left/right` here. That would also remove the `Span` as return type which is a bit problematic here since it encodes more information than just start/end points.